js/samples/ios/quickstart-swift/LaunchViewController.swift

```diff
@@ -58,6 +62,35 @@ class LaunchViewController: UIViewController {
         launchButton.addTarget(self, action: #selector(launchImmersiveReaderButton(sender:)), for: .touchUpInside)
         view.addSubview(launchButton)
 
+        disableGrammarButton = UIButton()
+        disableGrammarButton.backgroundColor = .darkGray
+        disableGrammarButton.contentEdgeInsets = UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10)
+        disableGrammarButton.setTitleColor(.white, for: .normal)
+        disableGrammarButton.setTitle("disableGrammar", for: .normal)
+        disableGrammarButton.addTarget(self, action: #selector(launchImmersiveReaderButton(sender:)), for: .touchUpInside)
+        view.addSubview(disableGrammarButton)
+        
+        disableTranslationButton = UIButton()
+        disableTranslationButton.backgroundColor = .darkGray
+        disableTranslationButton.contentEdgeInsets = UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10)
+        disableTranslationButton.setTitleColor(.white, for: .normal)
+        disableTranslationButton.setTitle("disableTranslation", for: .normal)
+        disableTranslationButton.addTarget(self, action: #selector(launchImmersiveReaderButton(sender:)), for: .touchUpInside)
+        view.addSubview(disableTranslationButton)
+        
+        disableLanguageDetectionButton = UIButton()
+        disableLanguageDetectionButton.backgroundColor = .darkGray
+        disableLanguageDetectionButton.contentEdgeInsets = UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10)
+        disableLanguageDetectionButton.setTitleColor(.white, for: .normal)
+        disableLanguageDetectionButton.setTitle("disableLanguageDetection", for: .normal)
+        disableLanguageDetectionButton.addTarget(self, action: #selector(launchImmersiveReaderButton(sender:)), for: .touchUpInside)
+        view.addSubview(disableLanguageDetectionButton)
+        
+        languageText = UITextField()
+        languageText.backgroundColor = .white
+        languageText.text = "en"
```

Comment:
is this always 'en' lang? How does that work with DisableLanguageDetection? Don't we usually give option to change the language on the chunk?

Reply:
that's only to set a default value to the TextField.

User is able to change the value in UI and that value is passed to the chunk
![image](https://user-images.githubusercontent.com/87383938/164916563-b3bd5bba-16da-499b-ae33-ed0b26901009.png)